Makefile

```diff
@@ -1,50 +1,27 @@
-# Shortcut targets
-default: test
+PACKAGE_NAME=github.com/projectcalico/libcalico-go
+GO_BUILD_VER=v0.27
 
-## Build binary
-all: test
+###############################################################################
+# Download and include Makefile.common before anything else
+###############################################################################
+MAKE_BRANCH?=$(GO_BUILD_VER)
+MAKE_REPO?=https://raw.githubusercontent.com/projectcalico/go-build/$(MAKE_BRANCH)
+
+Makefile.common: Makefile.common.$(MAKE_BRANCH)
+	cp "$<" "$@"
+Makefile.common.$(MAKE_BRANCH):
+	# Clean up any files downloaded from other branches so they don't accumulate.
+	rm -f Makefile.common.*
+	wget -nv $(MAKE_REPO)/Makefile.common -O "$@"
```

Comment:
It's debatable but I would imagine `curl` to have higher chances of being available :)
Either way, I suggest adding a target which checks for presence of `wget` or `curl` and then reference it as a dependency here.

Reply:
Fixed